config.yml

```diff
@@ -51,6 +51,9 @@ trading_options:
   # define in % when to take profit on a profitable coin.
   TAKE_PROFIT: .8
 
+  # Sell coin after a fixed duration in minutes. Value 0 means this fuctinality is disables.
+  SELL_AFTER_DURATION: 0
```

Comment:
Inherit 0s should not reflect disabled. Use something like
```yml
SELL_AFTER_TIME: True,False
SELL_AFTER_DURATION: 10
```

IMHO, this shouldnt be in this config or even in the binance script. It should be its own module which users can enable if they want...